src/components/30-organisms/footer/index.js

```diff
@@ -0,0 +1,279 @@
+// TODO fix that stuff
+/* eslint-disable import/no-extraneous-dependencies */
+import { CaretSvg } from '@axa-ch/materials/icons';
+import '@axa-ch/container';
+import { LitElement, html, svg, css, unsafeCSS } from 'lit-element';
+import defineOnce from '../../../utils/define-once';
+import styles from './index.scss';
+
+class AXAFooter extends LitElement {
+  static get tagName() {
+    return 'axa-footer';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      dynamic: { type: Boolean },
+    };
+  }
+
+  constructor() {
+    super();
+    this.dynamic = false;
+    this.onItemClick = () => {}; // Only for react
+    this._accordionActiveIndex = -1;
+  }
+
+  updated() {
+    this._removeEmptyListElements();
+  }
+
+  _removeEmptyListElements() {
+    this.shadowRoot
+      .querySelectorAll('.o-footer__main-content-panel-list-item')
+      .forEach(el => {
+        const label = el.querySelector('slot').assignedNodes()[0];
+        // Second part of IF-statement is an IE11 workaround
+        if (this._listElementHasNoContent(label)) {
+          el.style.display = 'none';
+        }
+      });
+  }
+
+  _listElementHasNoContent(label) {
```

Comment:
All helper methods without `this` don't belong into a class in my opinion - move to arrow function above class.

Reply:
Why that?
What about making them static? You probably have an issue with the performance aspect of that, right? `static` would resolve the performance problems.

The underscore notation was explicitly created to have private methods and properties. It would make me wonder if that is considered bad practice.

Reply:
But I am not sure. Maybe your approach would be more appropriate. Sounds somehow logical, but I never heard of a pattern like that.

Reply:
I went with the static approach, please have a look. :) Nice feedback!

Reply:
There is some confusion here in my opinion.

(Arrow) functions outside of class scope is just basic JS scoping rules, no 'best practice' needed at all. You have a module scope, use it!

Everything in a class that's non-static is added to the instance and thus consumes memory per  *instance*, so basic perf consideration.

"The underscore notation was explicitly created to have private methods and properties": citation needed.
There is nothing private here, just a convention by some people. You don't have privacy in JS.

And: I recommend to not use "I never heard of X".

Reply:
After the previous discussion, I will go with your approach.